modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -145,7 +145,8 @@ export default {
       triggerValidation: false,
       original: null,
       published: null,
-      errorCount: 0
+      errorCount: 0,
+      wasInTrash: false
```

Comment:
I'm bumping on the naming. `wasInTrash` isn't true for pages in their trash section and the "was" is very broad. `pieceInTrash`? From what I can tell, it's in the trash until it's saved, at which time the modal closes, so the past tense is no longer needed.

Reply:
I chose `wasInTrash` because `trash` would not get across why I need this at all (I need to know it *was in the trash from the start*). For pages this does not come up, one because we don't let you open page settings at all for trashed pages right now, and two because I put a guard on that anyway in case we do that later. The right way to take a page out of the trash is by dragging it out.

Reply:
I get all that about pages. That's why I'm suggesting `pieceInTrash`.